service-tests/json.js

```diff
@@ -35,7 +35,7 @@ t.create('JSON from uri | with prefix & suffix & label')
 
 t.create('JSON from uri | object doesnt exist')
   .get('.json?uri=https://github.com/badges/shields/raw/master/package.json&query=$.does_not_exist')
-  .expectJSON({ name: 'custom badge', value: '' });
+  .expectJSON({ name: 'custom badge', value: 'no result' });
```

Comment:
Want to test the color as well? @platan added a `_shields_test` format which exposes it.

Reply:
Nice, did not realise this was a thing!
Will update the tests